_SQLServer/TableLoadHeroes.sql

```diff
@@ -0,0 +1,478 @@
+'http://htkb.info/SDE/1.0/Cards/Heroes/Card_Hero_Princess_Malya_1.0_back.png', 
+'http://htkb.online/SDWiki/SDE/1.0/Cards/Heroes/Princess_Malya.xhtml', 
+'Hero', '1.0', '1.0 Core Set', 'Default',   
+'Even a racing super-star has time to sit down to a game of Super Dungeon Explore.  Princess Malya brings her love of speed into the Last Galaxy''s favorite online game, zipping through the dungeon at a breakneck pace.  Malya slays monsters with glee, but is always willing to lend fellow adventurers a hand with the help of some creation magic.');
+insert into Characters (CardIndex) VALUES (0);
+insert into KeywordAssignments (CharacterIndex, KeywordIndex) VALUES (1, 0); --! To look at later
+insert into AbilityAssignments (CharacterIndex, AbilityIndex) VALUES (0, 0); --Energy Blast
+insert into AbilityAssignments (CharacterIndex, AbilityIndex) VALUES (0, 0); --Purification
+insert into AbilityAssignments (CharacterIndex, AbilityIndex) VALUES (0, 0); --Mr. Tomn
+insert into ExploreCharacters (CharacterIndex, ModelSize, CreatureType, Movement, Actions, Strength, Armor, Will, Dexterity, Health, Potions) VALUES (0, 'Small', 'Human Hero', 8, 3, '1B1R', '2B', '2B', '2R', 5, 1);
+insert into OffenseAssignments (CharacterIndex, AttributeIndex, OffenseRange) VALUES (0,0,1);--! To look at later
+insert into DefenseAssignments (CharacterIndex, AttributeIndex) VALUES (0,4);--! To look at later
+
+
+--Princess Ruby
+insert into Cards (Name, PictureFront, PictureBack, Link, CardType, ProductSet, Module, Mode, Flavor) VALUES ('Princess Ruby', 
+'http://htkb.info/SDE/1.0/Cards/Heroes/Card_Hero_Princess_Ruby_1.0.png', 
+'http://htkb.info/SDE/1.0/Cards/Heroes/Card_Hero_Princess_Ruby_1.0_back.png', 
+'http://htkb.online/SDWiki/SDE/1.0/Cards/Heroes/Princess_Ruby.xhtml', 
+'Hero', '1.0', '1.0 Core Set', 'Default',   
+'The eldest of five sisters, Princess Ruby, is heiress to the royal throne of Crystalia.  Her kindness and wisdom are matched only by her stubbornness.  Despite her father''s protests and fears for her safety, she remains determined to travel her realm before she is required to rule it.');
+insert into Characters (CardIndex) VALUES (0);
+insert into KeywordAssignments (CharacterIndex, KeywordIndex) VALUES (1, 0); --! To look at later
+insert into AbilityAssignments (CharacterIndex, AbilityIndex) VALUES (0, 0); --Maiden's Favor
+insert into AbilityAssignments (CharacterIndex, AbilityIndex) VALUES (0, 0); --Maiden's Token
+insert into AbilityAssignments (CharacterIndex, AbilityIndex) VALUES (0, 0); --Maiden's Kiss
+insert into AbilityAssignments (CharacterIndex, AbilityIndex) VALUES (0, 0); --Happily Ever After
+insert into ExploreCharacters (CharacterIndex, ModelSize, CreatureType, Movement, Actions, Strength, Armor, Will, Dexterity, Health, Potions) VALUES (0, 'Small', 'Human Hero', 6, 3, '2B', '3B', '3B', '2B1R', 5, 1);
+insert into OffenseAssignments (CharacterIndex, AttributeIndex, OffenseRange) VALUES (0,3,1);--! To look at later
```

Comment:
OffenseRange should be 6

This character should have a 1st offense assignment for attack (0,0,1)